src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -693,6 +739,14 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
           handleSaveAttachment((MediaMmsMessageRecord)getSelectedMessageRecord());
           actionMode.finish();
           return true;
+        case R.id.menu_context_pin_message:
+					handlePinMessage(getSelectedMessageRecord());
+        	actionMode.finish();
+        	return true;
+			  case R.id.menu_context_unpin_message:
```

Comment:
Indentation on this case is weird, makes it hard to read

Reply:
yeah Lori has an issue with her Tabbing. I'll fix it.

Reply:
It looks better locally, my tab spacing was set up weird. I will attempt to fix this.